tools/smoke_test.py

```diff
@@ -0,0 +1,166 @@
+import os
```

Comment:
My opinion is that if its called test, it goes in test/, otherwise, its something else. If its not running on CI, and its under tools/, call it "verify_install.py" or something like that.